tests/functional/adapter/common.py

```diff
@@ -0,0 +1,136 @@
+from typing import Dict, Tuple, List
```

Comment:
typically something termed `common.py` would be used across the board or at least in multiple places. Do we intend to use this for other tests? 
If we don't, we should move this inside the `snapshot_tests` directory with a name like `table_operations.py` or something similar since it's tied to just this set of tests.

Reply:
Yes, the intention would be to use these across all of the tests. I would have called it `utils` if we didn't already have a directory called that. That directory is actually testing utilities, not utilities used for testing; so perhaps this directly should renamed `utils_tests` and this can be named `utils.py`.